swatch-core/schemas/billable_usage.yaml

```diff
@@ -44,7 +44,7 @@ properties:
       - Development/Test
       - Disaster Recovery
   uom:
```

Comment:
You are still using uom here. Causing the IQE tests to fail. Corresponding change needed below:
```
public BillableUsageAggregateKey(BillableUsage billableUsage) {
    this(
        billableUsage.getOrgId(),
        billableUsage.getProductId(),
        billableUsage.getUom(),
        Optional.ofNullable(billableUsage.getSla()).orElse(SlaEnum.EMPTY).value(),
        Optional.ofNullable(billableUsage.getUsage()).orElse(UsageEnum.EMPTY).value(),
        Optional.ofNullable(billableUsage.getBillingProvider())
            .map(BillingProviderEnum::value)
            .orElse(null),
        billableUsage.getBillingAccountId());
  }
```